simpleSequential/test_getIntegerListLength.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from getIntegerListLength import getIntegerListLength
+
+class TestGetIntegerListLength(unittest.TestCase):
+    def test_emptyListIntegerInput(self):
```

Comment:
Test name is inconsistent and not on point. In fact, I thought this one is the “test_isIntegerListEmpty.py” but turn out this is not. The test name is two long and I cannot understand what is the test case when I just read it. And I feel like the gramma is in correct.

Below is my recommendation:

1/ zeroIntegerListInput
2/ twoIntegerListInput

Short, on point, and consistent.